diff --git a/node_modules/react-native-fetch-sse/lib/commonjs/fetchSSE.js b/node_modules/react-native-fetch-sse/lib/commonjs/fetchSSE.js
index 00f5bc5..2569e02 100644
--- a/node_modules/react-native-fetch-sse/lib/commonjs/fetchSSE.js
+++ b/node_modules/react-native-fetch-sse/lib/commonjs/fetchSSE.js
@@ -35,8 +35,9 @@ async function fetchSSE(input, init, {
       return;
     }
     const stream = (0, _streams.consumeStream)(response, shouldParseJsonWhenOnMsg);
+    const reader = stream.getReader();
     let result = null;
-    while (!(result = await stream.getReader().read()).done) {
+    while (!(result = await reader.read()).done) {
       onMessage?.(result.value);
     }
 
diff --git a/node_modules/react-native-fetch-sse/lib/module/fetchSSE.js b/node_modules/react-native-fetch-sse/lib/module/fetchSSE.js
index dc9b8eb..9c4a086 100644
--- a/node_modules/react-native-fetch-sse/lib/module/fetchSSE.js
+++ b/node_modules/react-native-fetch-sse/lib/module/fetchSSE.js
@@ -28,8 +28,9 @@ export async function fetchSSE(input, init, {
       return;
     }
     const stream = consumeStream(response, shouldParseJsonWhenOnMsg);
+    const reader = stream.getReader();
     let result = null;
-    while (!(result = await stream.getReader().read()).done) {
+    while (!(result = await reader.read()).done) {
       onMessage?.(result.value);
     }
 
